tee-worker/litentry/core/data-providers/src/achainable.rs

```diff
@@ -625,6 +643,27 @@ impl Default for ParamsBasicTypeWithDatePercent {
 	}
 }
 
+// ParamsBasicTypeWithToken
+#[derive(Serialize, Deserialize, Debug, Clone)]
+#[serde(rename_all = "camelCase")]
+pub struct ParamsBasicTypeWithMirror {
+	#[serde(skip_serializing)]
+	#[serde(skip_deserializing)]
+	pub name: String,
+
+	pub chain: String,
+
+	#[serde(skip_serializing_if = "Option::is_none")]
+	pub post_quantity: Option<String>,
+}
+
+impl ParamsBasicTypeWithMirror {
+	pub fn new(name: String, network: &Web3Network, post_quantity: Option<String>) -> Self {
+		let chain = web3_network_to_chain(network);
```

Comment:
I don't think we need `web3_network_to_chain` anymore, it has derived from `IntoStaticStr`